sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -1642,6 +1653,9 @@ def get_transceiver_loopback(self):
         host_input_loopback_lane8   = BOOLEAN                          ; host side input loopback enable lane8
         ========================================================================
         """
+        loopback_support = self.get_transceiver_loopback_support()
+        if loopback_support is None:
+            return None
         trans_loopback = dict()
         trans_loopback['media_output_loopback'] = self.get_media_output_loopback()
         trans_loopback['media_input_loopback'] = self.get_media_input_loopback()
```

Comment:
if the loopack is not supported, return a list of 'N/A' for each lane.

Reply:
done